src/main/java/org/opensearch/timeseries/rest/handler/AbstractTimeSeriesActionHandler.java

```diff
@@ -907,11 +908,17 @@ protected void validateConfigFeatures(String id, boolean indexingDryRun, ActionL
                 String errorMessage;
                 if (isExceptionCausedByInvalidQuery(e)) {
                     errorMessage = CommonMessages.FEATURE_WITH_INVALID_QUERY_MSG + feature.getName();
+                    logger.error(errorMessage, e);
+                    multiFeatureQueriesResponseListener.onFailure(new OpenSearchStatusException(errorMessage, RestStatus.BAD_REQUEST, e));
                 } else {
                     errorMessage = CommonMessages.UNKNOWN_SEARCH_QUERY_EXCEPTION_MSG + feature.getName();
+                    logger.error(errorMessage, e);
+                    // If we see an unexpected error such as timeout or some task cancellation cause of search backpressure
+                    // we don't want to block detector creation as this is unlikely an error due to wrong configs
+                    // but we want to record what error was seen
```

Comment:
1. Are we sure the error in this block is related to timeout on the query on the OS cluster side?
2. It will be difficult to write IT for this change but we can still have UTs?
3. Why `(List.of(Optional.of(new double[] { 1.0 }` and not `Collections.singletonList(Optional.empty())` below?

Reply:
1. Yes we have tested this in production with both timeout and search backpressure blocks
2. I wrote a UT
3. CHanged to empty. either works here, its more that some results exists not the content